docs/contributor/CONTRIBUTING.md

```diff
@@ -25,12 +25,10 @@ There are a few basic ground-rules for contributors (including the maintainer(s)
 
 ### In General
 
-A Pull Request (PR) needs to be reviewed and approved by project maintainers.
-If a change does not alter any logic (e.g. comments, dependencies, docs), then it may be tagged
+* A Pull Request (PR) needs to be reviewed and approved by project maintainers.
+* If a change does not alter any logic (e.g. comments, dependencies, docs), then it may be tagged
 `A1-insubstantial` and merged faster.
```

Comment:
```suggestion
`R0-silent` and merged faster.
```

Reply:
Is it though? I thought `A1-insubstantial` is still relevan. If not, we should delete it (among other useless labels).

Reply:
Hm okay, looks like either `A1` or `R0` are required: https://github.com/paritytech/labels/blob/d4b233298f6cc54dc9edc9f0432ca98a40ab20ef/ruled_labels/specs_polkadot-sdk.yaml#L165  

But IMHO the overlap is pretty bit, since every `A1` can also be labeled `R0`. Maybe we should remove the `A1`.